recipe/meta.yaml

```
@@ -46,15 +51,10 @@ requirements:
     - cf-nvidia-tools 1                     # [linux]
   host:
     - cuda-version {{ cuda_compiler_version }}
-    - cudatoolkit  # [(cuda_compiler_version or "").startswith("11")]
-    - libcublas  # [(cuda_compiler_version or "").startswith("12")]
+    - libcublas  # [(cuda_compiler_version or "").startswith("12") or (cuda_compiler_version or "").startswith("13")]
```

So is this always true? Since we build with either CUDA 12 or 13?

```suggestion
    - libcublas
```

Also should we use `libcublas-dev` or do we prefer this for some reason?

---

what's the difference? we only need libcublas.so